_pages/en_US/installing-boot9strap-(slottool).txt

```diff
@@ -0,0 +1,61 @@
+---
+title: "Installing boot9strap (slotTool)"
+---
+
+### What You Need
+
+* The latest release of [Luma3DS](https://github.com/AuroraWright/Luma3DS/releases/latest)
+* The latest release of [unSAFE_MODE](https://github.com/zoogie/unSAFE_MODE/releases/latest)
+
+#### Section I  - slotTool
+1. Copy `boot.firm` and from the Luma3DS `.zip` to the root of your SD card
+1. Copy `slotTool.3dsx` from the unSAFE_MODE archive to the `3ds` folder and `usm.bin` to your sd root
```

Comment:
"sd root" -> to the root of your SD card

Reply:
It may be best to separate this into two steps to make things as clear as possible
1. Copy `slotTool.3dsx` from the unSAFE_MODE archive to the `/3ds/` folder (in line with other pages on the guide)
1. Copy `usm.bin` from the unSAFE_MODE archive to the root of your SD card

Reply:
Possibly rename `unSAFE_MODE archive` to RELEASE `.zip` to reduce ambiguity, in line with other pages on the guide:
`Copy boot9strap.firm and boot9strap.firm.sha from the boot9strap .zip `